collector/varnish/README.md

```diff
@@ -0,0 +1,63 @@
+---
+# Ingest metrics using the Varnish integration
+
+The OTEL Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
```

Comment:
Change occurences of OTel to OTEL. I see both out there, but we use OTel on our docs site.